docs/reference-guide/13-file-format.md

```diff
@@ -3,6 +3,14 @@ This is a list of all recent changes that came with new Antares Simulator featur
 
 ## v8.4.0
 ### Input
+#### Zero/infinite capacity for physical links only
+In file **settings/generaldata.ini**, in section `optimization`, add 2 values for existing property `transmission-capacities` [str]:
+
+* `false-physical` sets the capacity of physical links to 0
```

Comment:
I agree on everything. To avoid the prefix, we can use an **enum class**.

We should keep existing values in the read functions for compatibility
```cpp
if (key == "null-for-any-link" || key == "false") {
   prop = tnc::nullForAllLinks;
}
```